serverless/networking/serverless-ossm-tls.adoc

```diff
@@ -0,0 +1,160 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-tls"]
+= Configuring TLS for a custom domain using {ProductName}
+:context: serverless-ossm-tls
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+.Prerequisites
+* Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+* Install xref:../../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[{ProductName}].
+
+.Procedure
+. Create TLS certificates:
```

Comment:
Could you mention the following note or something similar?

> In the following example, we use `openssl` command but you can use your favorite tool to generate certificates.

There are many way to create certificates and we don't care how they were created.